SConstruct

```diff
@@ -2150,16 +2089,24 @@ elif env['OS'] == 'Darwin':
 else:
     env.PrependENVPath('LD_LIBRARY_PATH', Dir('#build/lib').abspath)
 
-for yaml in multi_glob(env, "data", "yaml"):
-    dest = Path() / "build" / "data" / yaml.name
-    build(env.Command(str(dest), yaml.path, Copy("$TARGET", "$SOURCE")))
+
+if (env["example_data"]
+    and not list(Path("data/example_data").glob("*.yaml"))
```

Comment:
This short circuits the checkout if any YAML files are in the folder... I wonder if we'll run into cases where out-of-date files are used. Can we make this only conditional on the build variable?

Reply:
The current check is similar to what we do for all of the other submodules -- we're really only testing that something has been checked out, and I don't think we want to run the `git` command for every incremental build. Once the submodule has been synced at least once, managing whether it's up-to-date is left up to the user, and can be seen by whether `git status` reports uncommitted changes in the working directory.